### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by rbromley10**
Reworded an awkward sounding sentence.

---
Checklist:
- [ ] Run automated tests (`make test`)
- [ ] Test end to end manually (e.g. rebuild registry/operator and run `cx refresh` in an example folder)
- [ ] Add license header to each new file
- [x] Update documentation
- [ ] Update examples and `cx init`
- [ ] Alert team if dev environment changed
- [ ] Cherry-pick into release branches if it's a bugfix
- [ ] Delete the branch once it's merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 **Machine learning platform for developers: build machine learning applications faster.**
 
-Cortex is an open-source platform for building, deploying, and managing machine learning applications in production. It is designed for any developer who wants to build machine learning powered services without having to worry about infrastructure challenges like configuring data pipelines, continuous deployment, and dependency management.
+Cortex is an open-source platform for building, deploying, and managing machine learning applications in production. It is designed for developers wanting to build services powered by machine learning without the worries of infrastructure challenges such as configuring data pipelines, continuous deplyment, and dependency management.
```
Comment (vishalbollu) [inline README.md]: I like the phrasing you suggested

nit: deplyment -> deployment

Comment (rbromley10) [reply README.md]: Fixed.

**Comment by vishalbollu** [comment]
@rbromley10 apologies for the delay. We are working towards setting up and following open source best practices. We appreciate your patience in the meanwhile.

**Comment by deliahu** [comment]
Hey @rbromley10, thanks again for your contribution! Our CLA agreement and process is finalized now, do you mind signing the updated CLA and then merging?

**Comment by rbromley10** [comment]
> Hey @rbromley10, thanks again for your contribution! Our CLA agreement and process is finalized now, do you mind signing the updated CLA and then merging?

Signed! I can't merge as I don't seem to have write access though.

**Comment by deliahu** [comment]
Ok I merged it, thank you!

**Pull request merged**